app/jobs/auth_trail/geocode_job.rb

```diff
@@ -10,12 +10,31 @@ def perform(login_activity)
         end
 
       if result
-        login_activity.update!(
```

Comment:
The changeset on this file is a bit large. Instead of checking if a column exists, I think it'd be cleaner to use `respond_to?`.

```ruby
attributes.each do |k, v|
  login_activity.send("#{k}=", v) if login_activity.respond_to?("#{k}=")
end
```